### ISSUE TITLE
[guide] use on viaproxy 3.3.0

### ISSUE THREAD

**Issue by GavinGoGaming** [ayunami2000/ayunViaProxyEagUtils#3]
TLDR: [download](https://github.com/GavinGoGaming/mediaology-game-repo/raw/main/eagle/ayunviaproxyutils/ayunViaProxyEagUtils-1.0-FOR-3.3.0.jar)

i compiled this with java 21, go cry about it
Tested on: hypixel (1.8.x), klash (1.21-1.21.1)

1. put attached file in plugins
2. run viaproxy-3.3.0.jar with java 21

if you are getting errors, try:
1. close viaproxy
2. go to vialoader/vpeautils.yml and disable eagler skins

For Hypixel, make sure to select version as `1.8.x`. If you get banned, dont complain, it legit warns you about that when you run viaproxy

download [here](https://github.com/GavinGoGaming/mediaology-game-repo/raw/main/eagle/ayunviaproxyutils/ayunViaProxyEagUtils-1.0-FOR-3.3.0.jar)
all of src is pretty much untouched, mainly just gradle and IJ project stuff

**Comment by GavinGoGaming** [comment]
also - this wont work with the java8 version if it isnt obvious enough.

**Comment by qwertyad1** [comment]
thank you, it was erroring out before.

**Comment by emburnev3** [comment]
wait a minute what plugins folder? do i need to host a server with viaproxy then put this in myself?

**Comment by GavinGoGaming** [comment]
> wait a minute what plugins folder? do i need to host a server with viaproxy then put this in myself?
> 

You have to download ViaProxy and run it, then put in this plugin. Then open viaproxy again and configure settings and start

Viaproxy isn't a server plugin

**Comment by colbster937** [comment]
https://github.com/WebMCDevelopment/ayunViaProxyEagUtils
